src/elements/Button/Button.styl

```diff
@@ -110,7 +125,7 @@
   background-color var(--button-background-color)
 
   padding $space-zero
-
+  
```

Comment:
Remove trailing spaces. Use a linter like ESlint for example.

Reply:
```suggestion 
padding $space-zero
border-top-width var(--button-top-border-width)
```